src/main/java/me/botsko/prism/commands/ReportCommand.java

```diff
@@ -12,273 +9,233 @@
+            HikariDataSource ds = (HikariDataSource) Prism.getPrismDataSource().getDataSource();
+
+            sender.sendMessage(Prism.messenger.playerMsg("Pool total: " + ChatColor.WHITE
+                    + ds.getHikariPoolMXBean().getTotalConnections()));
+            sender.sendMessage(Prism.messenger.playerMsg("Pool active: " + ChatColor.WHITE
+                    + ds.getHikariPoolMXBean().getActiveConnections()));
+            sender.sendMessage(Prism.messenger.playerMsg("Pool idle: " + ChatColor.WHITE
+                    + ds.getHikariPoolMXBean().getIdleConnections()));
+            sender.sendMessage(Prism.messenger.playerMsg("Pool min idle: " + ChatColor.WHITE
+                    + ds.getMinimumIdle()));
+            sender.sendMessage(Prism.messenger.playerMsg("Pool max idle: " + ChatColor.WHITE
+                    + ds.getMaximumPoolSize()));
+        }
+
+        boolean recorderActive = checkRecorderActive(plugin);
+
+        if (recorderActive) {
+            sender.sendMessage(Prism.messenger.playerSuccess("Recorder is currently queued or running!"));
+        } else {
+            sender.sendMessage(
+                    Prism.messenger.playerError("Recorder stopped running! DB conn problems? Try /pr recorder start"));
+        }
+
+        sender.sendMessage(Prism.messenger.playerSubduedHeaderMsg("Attempting to check connection readiness..."));
+
+        Connection conn = null;
+        try {
+
+            conn = Prism.getPrismDataSource().getConnection();
+            if (conn == null) {
+                sender.sendMessage(Prism.messenger.playerError("Pool returned NULL instead of a valid connection."));
+            } else if (conn.isClosed()) {
+                sender.sendMessage(Prism.messenger.playerError("Pool returned an already closed connection."));
+            } else if (conn.isValid(5)) {
+                sender.sendMessage(Prism.messenger.playerSuccess("Pool returned valid connection!"));
+            }
+        } catch (final SQLException e) {
+            sender.sendMessage(Prism.messenger.playerError("Error: " + e.getMessage()));
+            e.printStackTrace();
+        } finally {
+            if (conn != null)
+                try {
+                    conn.close();
+                } catch (final SQLException ignored) {
+                }
+        }
+    }
+
+
+    private void blockSumReports(final CallInfo call) {
+
+        // Process and validate all of the arguments
+        final QueryParameters parameters = PreprocessArgs.process(plugin, call.getSender(), call.getArgs(),
+                PrismProcessType.LOOKUP, 3, !plugin.getConfig().getBoolean("prism.queries.never-use-defaults"));
+        if (parameters == null) {
+            call.getSender()
+                    .sendMessage(Prism.messenger.playerError("You must specify parameters, at least one player."));
+            return;
+        }
+        // No actions
+        if(checkParams(parameters,call)){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [WhitespaceAround: '{' is not preceded with whitespace.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5438808)